samples/java_layout/src/androidTest/java/io/customer/android/sample/java_layout/ApplicationTest.java

```diff
@@ -0,0 +1,13 @@
+package io.customer.android.sample.java_layout;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+public class ApplicationTest {
```

Comment:
We should probably also move `MemoryLeakageInstrumentedTest` to Java sample from Kotlin at some point.